src/main/java/command/Command.java

```diff
@@ -0,0 +1,61 @@
+package command;
+
+import exception.DukeException;
+import storage.Storage;
+import task.TaskList;
+import ui.Response;
+import ui.Ui;
+
+/**
+ * Represents an abstraction of sub-types.
+ * A command is a parent to the different kinds of commands
+ * a user can execute - todo/deadline/event/delete/exit/incorrect/list/mark/unmark.
+ */
+public abstract class Command {
+
+    private static boolean isExit = false;
+    protected Response response;
+    /**
```

Comment:
Shouldn't there be a line separating the class variables from the method or its javadocs? 🤔